ZSH with global setopt shwordsplit produces an error to launch oh-my-posh print

### Code of Conduct

- [x] I agree to follow this project's Code of Conduct

### What happened?

Hello Oh-my-posh team.

I have a custom setup with some setopt flags in my `.zshrc`, and today i will try use the tooltips to improve the experience with the tool but i have an error with one of the setopt flags and oh-my-posh.

When I have set the setopt with the flag `shwordsplit` and the oh-my-posh enabled in the last line of the `.zshrc`
eval "$(oh-my-posh init zsh --config /home/test/t.omp.json)", every time that write a command with more than 2 words I have an error.

Example:
```
❯ one twoError: accepts 1 arg(s), received 2
Usage:
  oh-my-posh print [debug|primary|secondary|transient|right|tooltip|valid|error|preview] [flags]

Flags:
      --cleared                do we have a clear terminal or not
      --column int             the column position of the cursor
      --command string         tooltip command
      --escape                 escape the ANSI sequences for the shell (default true)
      --eval                   output the prompt for eval
      --execution-time float   timing of the last command
  -f, --force                  force rendering the segments
  -h, --help                   help for print
      --job-count int          number of background jobs
      --no-status              no valid status code (cancelled or no command yet)
      --pipestatus string      the PIPESTATUS array
  -p, --plain                  plain text output (no ANSI)
      --pswd string            current working directory (according to pwsh)
      --pwd string             current working directory
      --shell string           the shell to print for
      --shell-version string   the shell version
  -s, --stack-count int        number of locations on the stack
      --status int             last known status code
  -w, --terminal-width int     width of the terminal

Global Flags:
  -c, --config string   config file path
```

### Theme

{
  "$schema": "https://raw.githubusercontent.com/JanDeDobbeleer/oh-my-posh/main/themes/schema.json",
  "blocks": [
    {
      "alignment": "left",
      "segments": [
        {
          "background": "#d3d7cf",
          "foreground": "#000000",
          "style": "diamond",
          "template": " {{ if .WSL }}WSL at {{ end }}{{.Icon}} ",
          "type": "os"
        },
        {
          "background": "#3465a4",
          "foreground": "#e4e4e4",
          "powerline_symbol": "\ue0b0",
          "properties": {
            "home_icon": "~",
            "style": "full"
          },
          "style": "powerline",
          "template": " \uf07c {{ .Path }} ",
          "type": "path"
        },
        {
          "background": "#4e9a06",
          "background_templates": [
            "{{ if or (.Working.Changed) (.Staging.Changed) }}#c4a000{{ end }}",
            "{{ if and (gt .Ahead 0) (gt .Behind 0) }}#f26d50{{ end }}",
            "{{ if gt .Ahead 0 }}#89d1dc{{ end }}",
            "{{ if gt .Behind 0 }}#4e9a06{{ end }}"
          ],
          "foreground": "#000000",
          "powerline_symbol": "\ue0b0",
          "properties": {
            "branch_icon": "\uf126 ",
            "fetch_stash_count": true,
            "fetch_status": true,
            "fetch_upstream_icon": true
          },
          "style": "powerline",
          "template": " {{ .UpstreamIcon }}{{ .HEAD }}{{if .BranchStatus }} {{ .BranchStatus }}{{ end }}{{ if .Working.Changed }} \uf044 {{ .Working.String }}{{ end }}{{ if and (.Working.Changed) (.Staging.Changed) }} |{{ end }}{{ if .Staging.Changed }} \uf046 {{ .Staging.String }}{{ end }}{{ if gt .StashCount 0 }} \ueb4b {{ .StashCount }}{{ end }} ",
          "type": "git"
        }
      ],
      "type": "prompt"
    },
    {
      "alignment": "right",
      "segments": [
        {
          "background": "#689f63",
          "foreground": "#ffffff",
          "invert_powerline": true,
          "powerline_symbol": "\ue0b2",
          "properties": {
            "fetch_version": true
          },
          "style": "powerline",
          "template": " {{ if .PackageManagerIcon }}{{ .PackageManagerIcon }} {{ end }}{{ .Full }} \ue718 ",
          "type": "node"
        },
        {
          "background": "#00acd7",
          "foreground": "#111111",
          "invert_powerline": true,
          "powerline_symbol": "\ue0b2",
          "properties": {
            "fetch_version": true
          },
          "style": "powerline",
          "template": " {{ if .Error }}{{ .Error }}{{ else }}{{ .Full }}{{ end }} \ue627 ",
          "type": "go"
        },
        {
          "background": "#4063D8",
          "foreground": "#111111",
          "invert_powerline": true,
          "powerline_symbol": "\ue0b2",
          "properties": {
            "fetch_version": true
          },
          "style": "powerline",
          "template": " {{ if .Error }}{{ .Error }}{{ else }}{{ .Full }}{{ end }} \ue624 ",
          "type": "julia"
        },
        {
          "background": "#FFDE57",
          "foreground": "#111111",
          "invert_powerline": true,
          "powerline_symbol": "\ue0b2",
          "properties": {
            "display_mode": "files",
            "fetch_virtual_env": false
          },
          "style": "powerline",
          "template": " {{ if .Error }}{{ .Error }}{{ else }}{{ .Full }}{{ end }} \ue235 ",
          "type": "python"
        },
        {
          "background": "#AE1401",
          "foreground": "#ffffff",
          "invert_powerline": true,
          "powerline_symbol": "\ue0b2",
          "properties": {
            "display_mode": "files",
            "fetch_version": true
          },
          "style": "powerline",
          "template": " {{ if .Error }}{{ .Error }}{{ else }}{{ .Full }}{{ end }} \ue791 ",
          "type": "ruby"
        },
        {
          "background": "#FEAC19",
          "foreground": "#ffffff",
          "invert_powerline": true,
          "powerline_symbol": "\ue0b2",
          "properties": {
            "display_mode": "files",
            "fetch_version": false
          },
          "style": "powerline",
          "template": " {{ if .Error }}{{ .Error }}{{ else }}{{ .Full }}{{ end }} \uf0e7",
          "type": "azfunc"
        },
        {
          "background_templates": [
            "{{if contains \"default\" .Profile}}#FFA400{{end}}",
            "{{if contains \"jan\" .Profile}}#f1184c{{end}}"
          ],
          "foreground": "#ffffff",
          "invert_powerline": true,
          "powerline_symbol": "\ue0b2",
          "properties": {
            "display_default": false
          },
          "style": "powerline",
          "template": " {{ .Profile }}{{ if .Region }}@{{ .Region }}{{ end }} \ue7ad ",
          "type": "aws"
        },
        {
          "background": "#ffff66",
          "foreground": "#111111",
          "invert_powerline": true,
          "powerline_symbol": "\ue0b2",
          "style": "powerline",
          "template": " \uf0ad ",
          "type": "root"
        },
        {
          "background": "#c4a000",
          "foreground": "#000000",
          "invert_powerline": true,
          "powerline_symbol": "\ue0b2",
          "style": "powerline",
          "template": " {{ .FormattedMs }} \uf252 ",
          "type": "executiontime"
        },
        {
          "background": "#000000",
          "background_templates": [
            "{{ if gt .Code 0 }}#cc2222{{ end }}"
          ],
          "foreground": "#d3d7cf",
          "invert_powerline": true,
          "powerline_symbol": "\ue0b2",
          "properties": {
            "always_enabled": true
          },
          "style": "powerline",
          "template": " {{ if gt .Code 0 }}{{ reason .Code }}{{ else }}\uf42e{{ end }} ",
          "type": "status"
        }
      ],
      "type": "prompt"
    },
    {
      "alignment": "left",
      "newline": true,
      "segments": [
        {
          "foreground": "#73e230",
          "style": "plain",
          "template": "❯",
          "type": "text"
        }
      ],
      "type": "prompt"
    }
  ],
  "transient_prompt": {
    "background": "transparent",
    "foreground": "#73e230",
    "template": "❯"
  },
  "tooltips": [
    {
      "type": "git",
      "tips": [
        "git",
        "g"
      ],
      "style": "diamond",
      "foreground": "#193549",
      "background": "#fffb38",
      "leading_diamond": "",
      "trailing_diamond": "",
      "template": "{{ .HEAD }}{{ if .Staging.Changed }}  {{ .Staging.String }}{{ end }}{{ if and (.Working.Changed) (.Staging.Changed) }} |{{ end }}{{ if .Working.Changed }}  {{ .Working.String }}{{ end }}",
      "properties": {
        "fetch_status": true,
        "fetch_upstream_icon": true
      }
    }
  ],
  "console_title_template": "{{ .Shell }} in {{ .Folder }}",
  "final_space": true,
  "version": 3
}


### What OS are you seeing the problem on?

Linux

### Which shell are you using?

zsh

### Log output

```Shell
❯oh-my-posh debug --plain

Version: 26.14.3

Shell: zsh (5.9)

Prompt:

    ~                                                                                                                               SOFTWARE 
❯ 

Segments:

ConsoleTitle(true)                         -   0 ms
Os(true)                                   -   0 ms
Path(true)                                 -   1 ms
Git(false)                                 -   0 ms
Node(false)                                -   0 ms
Go(false)                                  -   0 ms
Julia(false)                               -   0 ms
Python(false)                              -   0 ms
Ruby(false)                                -   0 ms
Azfunc(false)                              -   0 ms
Aws(false)                                 -   0 ms
Root(false)                                -   0 ms
Executiontime(false)                       -   0 ms
Status(true)                               -   0 ms
Text(true)                                 -   0 ms

Run duration: 20.392178ms

Cache path: /home/test/.cache/oh-my-posh

Config path: /home/test/t.omp.json

Logs:

[DEBUG] 21:32:32.315 debug.go:35 → logging enabled, raw mode: true
[DEBUG] 21:32:32.315 load.go:filePath:102 → config set using POSH_THEME: /home/test/t.omp.json
[TRACE] 21:32:32.315 load.go:filePath() - 12.266µs
[TRACE] 21:32:32.316 load.go:readConfig() - 780.96µs
[TRACE] 21:32:32.316 load.go:Load() - 806.125µs
[DEBUG] 21:32:32.316 home.go:12 → /home/test
[TRACE] 21:32:32.316 path.go:Path() - 22.322µs
[DEBUG] 21:32:32.316 file.go:Init:26 → loading cache file: /home/test/.cache/oh-my-posh/omp.cache
[DEBUG] 21:32:32.317 file.go:Init:47 → loading cache key: font_list_cache
[DEBUG] 21:32:32.317 file.go:Init:47 → loading cache key: https://github.com/ryanoasis/nerd-fonts/releases/download/v3.4.0/Meslo.zip
[DEBUG] 21:32:32.317 file.go:Init:47 → loading cache key: is_wsl
[DEBUG] 21:32:32.317 file.go:Init:47 → loading cache key: INITVERSIONZSH
[DEBUG] 21:32:32.317 file.go:Init:47 → loading cache key: environment_platform
[TRACE] 21:32:32.317 file.go:Init(/home/test/.cache/oh-my-posh/omp.cache) - 131.895µs
[DEBUG] 21:32:32.317 home.go:12 → /home/test
[TRACE] 21:32:32.317 path.go:Path() - 8.225µs
[DEBUG] 21:32:32.317 file.go:Init:26 → loading cache file: /home/test/.cache/oh-my-posh/omp.cache.f52c3a3c-6671-4111-9fb1-d49f03141042
[DEBUG] 21:32:32.317 file.go:Init:47 → loading cache key: prompt_count_cache
[DEBUG] 21:32:32.317 file.go:Init:47 → loading cache key: template_cache
[TRACE] 21:32:32.317 file.go:Init(/home/test/.cache/oh-my-posh/omp.cache.f52c3a3c-6671-4111-9fb1-d49f03141042) - 21.857µs
[DEBUG] 21:32:32.317 file.go:Get:80 → getting cache key: prompt_count_cache with value: 4
[TRACE] 21:32:32.317 terminal.go:setPromptCount() - 3.571µs
[TRACE] 21:32:32.317 terminal.go:GOOS() - 115ns
[DEBUG] 21:32:32.317 terminal.go:setPwd:115 → /home/test
[TRACE] 21:32:32.317 terminal.go:setPwd() - 7.427µs
[TRACE] 21:32:32.317 terminal.go:Init() - 711.618µs
[TRACE] 21:32:32.317 terminal.go:Shell() - 71ns
[TRACE] 21:32:32.317 terminal.go:Flags() - 130ns
[DEBUG] 21:32:32.317 file.go:Get:80 → getting cache key: template_cache with value: {"SegmentsCache":{"Os":{"Segment":{"Text":"  ","Index":0},"Icon":""},"Path":{"Segment":{"Text":"  ~ ","Index":1},"Location":"/home/test","Path":"~","Folders":[],"StackCount":0,"Writable":true,"RootDir":false},"Status":{"Segment":{"Text":" SOFTWARE ","Index":0},"String":"70","Meaning":"SOFTWARE","Error":true},"Text":{"Segment":{"Text":"❯","Index":0}}},"Segments":{},"Var":{},"PWD":"~","Folder":"test","PSWD":"","UserName":"test","HostName":"test-VirtualBox","ShellVersion":"5.9","Shell":"zsh","AbsolutePWD":"/home/test","OS":"ubuntu","Version":"26.14.3","PromptCount":4,"SHLVL":2,"Jobs":0,"Code":70,"WSL":false,"Root":false}
[TRACE] 21:32:32.317 cache.go:restoreCache() - 206.85µs
[DEBUG] 21:32:32.317 writer.go:Init:106 → terminal program: Unknown
[DEBUG] 21:32:32.317 writer.go:Init:107 → terminal shell: shell
[DEBUG] 21:32:32.317 terminal.go:Getenv:77 → NO DATA
[TRACE] 21:32:32.317 terminal.go:Getenv(OMP_CACHE_DISABLED) - 5.585µs
[DEBUG] 21:32:32.317 file.go:Get:75 → cache key not found: accent_color
[TRACE] 21:32:32.317 colors.go:SetAccentColor() - 5.096µs
[TRACE] 21:32:32.317 terminal.go:Shell() - 71ns
[DEBUG] 21:32:32.317 terminal.go:Getenv:77 → 5.9
[TRACE] 21:32:32.317 terminal.go:Getenv(POSH_SHELL_VERSION) - 2.091µs
[DEBUG] 21:32:32.317 debug.go:PrintDebug:24 → segment: Title
[DEBUG] 21:32:32.317 text.go:patchTemplate:125 → {{ .Shell }} in {{ .Folder }}
[TRACE] 21:32:32.318 text.go:Render({{ .Shell }} in {{ .Folder }}) - 107.549µs
[TRACE] 21:32:32.318 terminal.go:Flags() - 30ns
[DEBUG] 21:32:32.318 terminal.go:Getenv:77 → NO DATA
[TRACE] 21:32:32.318 terminal.go:Getenv(POSH_CURSOR_LINE) - 1.702µs
[DEBUG] 21:32:32.318 terminal.go:Getenv:77 → NO DATA
[TRACE] 21:32:32.318 terminal.go:Getenv(POSH_CURSOR_COLUMN) - 1.211µs
[TRACE] 21:32:32.318 terminal.go:Flags() - 19ns
[TRACE] 21:32:32.318 terminal.go:Flags() - 18ns
[TRACE] 21:32:32.318 terminal.go:Flags() - 66ns
[DEBUG] 21:32:32.318 home.go:12 → /home/test
[TRACE] 21:32:32.318 terminal.go:GOOS() - 58ns
[DEBUG] 21:32:32.318 segment.go:Execute:113 → segment: Git
[DEBUG] 21:32:32.318 file.go:Get:75 → cache key not found: toggle_cache
[DEBUG] 21:32:32.318 segment.go:isToggled:234 → no toggles found
[DEBUG] 21:32:32.318 terminal.go:ResolveSymlink:208 → /home/test
[TRACE] 21:32:32.318 terminal.go:ResolveSymlink(/home/test) - 12.415µs
[ERROR] 21:32:32.318 terminal.go:HasParentFilePath:477 → stat .git: no such file or directory
[TRACE] 21:32:32.318 terminal.go:HasParentFilePath(.git) - 141.213µs
[TRACE] 21:32:32.318 terminal.go:Flags() - 257ns
[TRACE] 21:32:32.318 terminal.go:Flags() - 200ns
[DEBUG] 21:32:32.318 home.go:12 → /home/test
[TRACE] 21:32:32.318 terminal.go:GOOS() - 81ns
[DEBUG] 21:32:32.318 segment.go:Execute:113 → segment: Os
[DEBUG] 21:32:32.318 file.go:Get:75 → cache key not found: toggle_cache
[DEBUG] 21:32:32.318 home.go:12 → /home/test
[DEBUG] 21:32:32.318 segment.go:isToggled:234 → no toggles found
[TRACE] 21:32:32.318 terminal.go:GOOS() - 57ns
[DEBUG] 21:32:32.318 segment.go:Execute:113 → segment: Path
[TRACE] 21:32:32.318 terminal.go:GOOS() - 20ns
[DEBUG] 21:32:32.318 file.go:Get:75 → cache key not found: toggle_cache
[DEBUG] 21:32:32.318 segment.go:isToggled:234 → no toggles found
[DEBUG] 21:32:32.318 file.go:Get:80 → getting cache key: environment_platform with value: ubuntu
[DEBUG] 21:32:32.318 terminal_unix.go:Platform:92 → ubuntu
[DEBUG] 21:32:32.318 properties.go:GetBool:22 → display_distro_name: false
[DEBUG] 21:32:32.318 properties.go:GetBool:22 → display_cygpath: false
[TRACE] 21:32:32.318 terminal.go:GOOS() - 20ns
[TRACE] 21:32:32.318 separator.go:Separator() - 33ns
[TRACE] 21:32:32.318 terminal.go:Shell() - 89ns
[TRACE] 21:32:32.318 terminal.go:Shell() - 21ns
[DEBUG] 21:32:32.318 properties.go:GetString:28 → ubuntu: 
[TRACE] 21:32:32.318 terminal.go:GOOS() - 34ns
[DEBUG] 21:32:32.318 properties.go:GetBool:22 → mapped_locations_enabled: true
[DEBUG] 21:32:32.318 properties.go:GetString:28 → windows_registry_icon: 
[DEBUG] 21:32:32.318 properties.go:GetString:28 → windows_registry_icon: 
[DEBUG] 21:32:32.318 home.go:12 → /home/test
[TRACE] 21:32:32.318 separator.go:Separator() - 49ns
[TRACE] 21:32:32.318 terminal.go:GOOS() - 37ns
[TRACE] 21:32:32.318 terminal.go:GOOS() - 24ns
[DEBUG] 21:32:32.318 properties.go:GetString:28 → home_icon: ~
[DEBUG] 21:32:32.318 properties.go:GetKeyValueMap:46 → mapped_locations: map[]
[TRACE] 21:32:32.318 separator.go:Separator() - 22ns
[TRACE] 21:32:32.318 terminal.go:GOOS() - 25ns
[TRACE] 21:32:32.318 terminal.go:GOOS() - 19ns
[TRACE] 21:32:32.318 separator.go:Separator() - 19ns
[TRACE] 21:32:32.318 terminal.go:GOOS() - 33ns
[TRACE] 21:32:32.318 terminal.go:GOOS() - 19ns
[DEBUG] 21:32:32.318 text.go:patchTemplate:125 →  {{ if .WSL }}WSL at {{ end }}{{.Data.Icon}} 
[TRACE] 21:32:32.318 terminal.go:GOOS() - 29ns
[TRACE] 21:32:32.318 terminal.go:Flags() - 41ns
[DEBUG] 21:32:32.319 properties.go:GetStringArray:52 → cycle: []
[DEBUG] 21:32:32.319 properties.go:GetString:28 → folder_separator_template: 
[DEBUG] 21:32:32.319 properties.go:GetString:28 → folder_separator_icon: /
[DEBUG] 21:32:32.319 properties.go:GetBool:22 → cycle_folder_separator: false
[DEBUG] 21:32:32.319 properties.go:GetString:28 → folder_format: %s
[DEBUG] 21:32:32.319 properties.go:GetString:28 → edge_format: %s
[DEBUG] 21:32:32.320 properties.go:GetString:28 → left_format: %s
[DEBUG] 21:32:32.320 properties.go:GetString:28 → right_format: %s
[TRACE] 21:32:32.320 terminal.go:Flags() - 143ns
[TRACE] 21:32:32.320 terminal.go:GOOS() - 45ns
[TRACE] 21:32:32.320 terminal.go:StackCount() - 48ns
[TRACE] 21:32:32.320 terminal_unix.go:DirIsWritable(/home/test) - 5.429µs
[TRACE] 21:32:32.320 text.go:Render( {{ if .WSL }}WSL at {{ end }}{{.Icon}} ) - 1.69288ms
[TRACE] 21:32:32.320 text.go:Render(diamond) - 84ns
[DEBUG] 21:32:32.321 text.go:patchTemplate:125 →   {{ .Data.Path }} 
[TRACE] 21:32:32.322 text.go:Render(  {{ .Path }} ) - 298.699µs
[TRACE] 21:32:32.322 text.go:Render(powerline) - 107ns
[TRACE] 21:32:32.322 text.go:Render(powerline) - 78ns
[TRACE] 21:32:32.322 terminal.go:Shell() - 96ns
[TRACE] 21:32:32.322 terminal.go:Shell() - 21ns
[TRACE] 21:32:32.322 terminal.go:Flags() - 78ns
[DEBUG] 21:32:32.322 home.go:12 → /home/test
[TRACE] 21:32:32.322 terminal.go:GOOS() - 38ns
[DEBUG] 21:32:32.322 segment.go:Execute:113 → segment: Status
[DEBUG] 21:32:32.322 file.go:Get:75 → cache key not found: toggle_cache
[DEBUG] 21:32:32.322 segment.go:isToggled:234 → no toggles found
[TRACE] 21:32:32.322 terminal.go:StatusCodes() - 61ns
[DEBUG] 21:32:32.322 properties.go:GetString:28 → status_template: {{ .Code }}
[DEBUG] 21:32:32.322 text.go:patchTemplate:125 → {{ .Code }}
[TRACE] 21:32:32.322 text.go:Render({{ .Code }}) - 30.69µs
[DEBUG] 21:32:32.322 properties.go:GetBool:22 → always_enabled: true
[TRACE] 21:32:32.322 terminal.go:Flags() - 21ns
[DEBUG] 21:32:32.322 home.go:12 → /home/test
[TRACE] 21:32:32.322 terminal.go:GOOS() - 35ns
[DEBUG] 21:32:32.322 segment.go:Execute:113 → segment: Node
[DEBUG] 21:32:32.322 file.go:Get:75 → cache key not found: toggle_cache
[DEBUG] 21:32:32.322 segment.go:isToggled:234 → no toggles found
[DEBUG] 21:32:32.322 properties.go:GetStringArray:52 → extensions: [*.js *.ts package.json .nvmrc pnpm-workspace.yaml .pnpmfile.cjs .vue]
[DEBUG] 21:32:32.322 properties.go:GetStringArray:52 → folders: []
[DEBUG] 21:32:32.322 properties.go:GetBool:22 → home_enabled: false
[DEBUG] 21:32:32.322 home.go:12 → /home/test
[TRACE] 21:32:32.322 text.go:Render(powerline) - 257ns
[TRACE] 21:32:32.322 terminal.go:Flags() - 19ns
[DEBUG] 21:32:32.322 home.go:12 → /home/test
[TRACE] 21:32:32.322 terminal.go:GOOS() - 46ns
[DEBUG] 21:32:32.322 segment.go:Execute:113 → segment: Go
[DEBUG] 21:32:32.322 file.go:Get:75 → cache key not found: toggle_cache
[DEBUG] 21:32:32.322 segment.go:isToggled:234 → no toggles found
[DEBUG] 21:32:32.322 properties.go:GetStringArray:52 → extensions: [*.go go.mod go.sum go.work go.work.sum]
[DEBUG] 21:32:32.322 properties.go:GetStringArray:52 → folders: []
[DEBUG] 21:32:32.322 properties.go:GetBool:22 → home_enabled: false
[DEBUG] 21:32:32.322 home.go:12 → /home/test
[TRACE] 21:32:32.322 terminal.go:Flags() - 34ns
[TRACE] 21:32:32.322 text.go:Render(powerline) - 112ns
[DEBUG] 21:32:32.322 home.go:12 → /home/test
[TRACE] 21:32:32.322 terminal.go:Flags() - 135ns
[TRACE] 21:32:32.322 terminal.go:GOOS() - 37ns
[DEBUG] 21:32:32.322 segment.go:Execute:113 → segment: Aws
[DEBUG] 21:32:32.322 file.go:Get:75 → cache key not found: toggle_cache
[DEBUG] 21:32:32.322 segment.go:isToggled:234 → no toggles found
[DEBUG] 21:32:32.322 home.go:12 → /home/test
[DEBUG] 21:32:32.322 properties.go:GetBool:22 → display_default: false
[TRACE] 21:32:32.322 terminal.go:GOOS() - 42ns
[DEBUG] 21:32:32.322 terminal.go:Getenv:77 → NO DATA
[TRACE] 21:32:32.322 terminal.go:Getenv(AWS_VAULT) - 2.162µs
[DEBUG] 21:32:32.322 segment.go:Execute:113 → segment: Python
[DEBUG] 21:32:32.322 terminal.go:Getenv:77 → NO DATA
[TRACE] 21:32:32.322 terminal.go:Getenv(AWS_DEFAULT_PROFILE) - 1.536µs
[DEBUG] 21:32:32.322 file.go:Get:75 → cache key not found: toggle_cache
[DEBUG] 21:32:32.322 terminal.go:Getenv:77 → NO DATA
[DEBUG] 21:32:32.322 segment.go:isToggled:234 → no toggles found
[TRACE] 21:32:32.322 terminal.go:Getenv(AWS_PROFILE) - 1.121µs
[DEBUG] 21:32:32.322 terminal.go:Getenv:77 → NO DATA
[TRACE] 21:32:32.322 terminal.go:Getenv(AWS_REGION) - 1.29µs
[DEBUG] 21:32:32.322 terminal.go:Getenv:77 → NO DATA
[DEBUG] 21:32:32.322 properties.go:GetString:28 → display_mode: files
[TRACE] 21:32:32.322 terminal.go:Getenv(AWS_DEFAULT_REGION) - 1.091µs
[DEBUG] 21:32:32.322 terminal.go:Getenv:77 → NO DATA
[TRACE] 21:32:32.322 terminal.go:Getenv(AWS_CONFIG_FILE) - 1.256µs
[DEBUG] 21:32:32.322 home.go:12 → /home/test
[DEBUG] 21:32:32.322 properties.go:GetStringArray:52 → extensions: [*.py *.ipynb pyproject.toml venv.bak]
[DEBUG] 21:32:32.322 properties.go:GetStringArray:52 → folders: [.venv venv virtualenv venv-win pyenv-win]
[DEBUG] 21:32:32.322 properties.go:GetBool:22 → home_enabled: false
[DEBUG] 21:32:32.322 home.go:12 → /home/test
[ERROR] 21:32:32.322 terminal.go:FileContent:220 → open /home/test/.aws/config: no such file or directory
[TRACE] 21:32:32.322 terminal.go:Flags() - 174ns
[TRACE] 21:32:32.322 terminal.go:FileContent(/home/test/.aws/config) - 9.013µs
[TRACE] 21:32:32.322 terminal.go:Flags() - 67ns
[DEBUG] 21:32:32.322 home.go:12 → /home/test
[TRACE] 21:32:32.322 terminal.go:GOOS() - 34ns
[DEBUG] 21:32:32.322 segment.go:Execute:113 → segment: Julia
[DEBUG] 21:32:32.322 home.go:12 → /home/test
[DEBUG] 21:32:32.322 file.go:Get:75 → cache key not found: toggle_cache
[TRACE] 21:32:32.322 terminal.go:GOOS() - 73ns
[DEBUG] 21:32:32.322 segment.go:isToggled:234 → no toggles found
[DEBUG] 21:32:32.322 home.go:12 → /home/test
[TRACE] 21:32:32.322 terminal.go:GOOS() - 33ns
[DEBUG] 21:32:32.322 properties.go:GetStringArray:52 → extensions: [*.jl]
[DEBUG] 21:32:32.322 segment.go:Execute:113 → segment: Root
[DEBUG] 21:32:32.322 file.go:Get:75 → cache key not found: toggle_cache
[DEBUG] 21:32:32.322 properties.go:GetStringArray:52 → folders: []
[DEBUG] 21:32:32.322 file.go:Get:75 → cache key not found: toggle_cache
[DEBUG] 21:32:32.322 properties.go:GetBool:22 → home_enabled: false
[DEBUG] 21:32:32.322 segment.go:isToggled:234 → no toggles found
[DEBUG] 21:32:32.322 home.go:12 → /home/test
[TRACE] 21:32:32.322 text.go:Render(powerline) - 56ns
[TRACE] 21:32:32.322 terminal_unix.go:Root() - 407ns
[TRACE] 21:32:32.322 terminal.go:Flags() - 19ns
[TRACE] 21:32:32.322 text.go:Render(powerline) - 68ns
[DEBUG] 21:32:32.322 home.go:12 → /home/test
[TRACE] 21:32:32.322 terminal.go:Flags() - 19ns
[TRACE] 21:32:32.322 terminal.go:GOOS() - 19ns
[DEBUG] 21:32:32.322 home.go:12 → /home/test
[DEBUG] 21:32:32.322 segment.go:Execute:113 → segment: Ruby
[DEBUG] 21:32:32.322 properties.go:GetStringArray:52 → extensions: [host.json local.settings.json function.json]
[TRACE] 21:32:32.322 terminal.go:GOOS() - 19ns
[DEBUG] 21:32:32.322 file.go:Get:75 → cache key not found: toggle_cache
[DEBUG] 21:32:32.322 properties.go:GetStringArray:52 → folders: []
[DEBUG] 21:32:32.322 segment.go:Execute:113 → segment: Executiontime
[DEBUG] 21:32:32.322 segment.go:isToggled:234 → no toggles found
[DEBUG] 21:32:32.322 file.go:Get:75 → cache key not found: toggle_cache
[DEBUG] 21:32:32.322 properties.go:GetBool:22 → home_enabled: false
[DEBUG] 21:32:32.322 segment.go:isToggled:234 → no toggles found
[DEBUG] 21:32:32.322 home.go:12 → /home/test
[DEBUG] 21:32:32.322 properties.go:GetStringArray:52 → extensions: [*.rb Rakefile Gemfile]
[DEBUG] 21:32:32.322 properties.go:GetBool:22 → always_enabled: false
[DEBUG] 21:32:32.322 properties.go:GetStringArray:52 → folders: []
[TRACE] 21:32:32.322 terminal.go:ExecutionTime() - 41ns
[DEBUG] 21:32:32.322 properties.go:GetBool:22 → home_enabled: false
[DEBUG] 21:32:32.322 properties.go:GetFloat64:34 → threshold: 500.000000
[DEBUG] 21:32:32.322 home.go:12 → /home/test
[TRACE] 21:32:32.322 text.go:Render(powerline) - 77ns
[TRACE] 21:32:32.322 text.go:Render(powerline) - 53ns
[TRACE] 21:32:32.322 text.go:Render(powerline) - 26ns
[TRACE] 21:32:32.322 text.go:Render(powerline) - 28ns
[TRACE] 21:32:32.322 text.go:Render(powerline) - 25ns
[DEBUG] 21:32:32.322 text.go:patchTemplate:125 →  {{ if gt .Code 0 }}{{ reason .Code }}{{ else }}{{ end }} 
[TRACE] 21:32:32.322 text.go:Render( {{ if gt .Code 0 }}{{ reason .Code }}{{ else }}{{ end }} ) - 244.162µs
[DEBUG] 21:32:32.322 text.go:patchTemplate:125 → {{ if gt .Code 0 }}#cc2222{{ end }}
[TRACE] 21:32:32.322 text.go:Render({{ if gt .Code 0 }}#cc2222{{ end }}) - 12.579µs
[TRACE] 21:32:32.322 text.go:Render(powerline) - 59ns
[DEBUG] 21:32:32.322 text.go:patchTemplate:125 → {{ if gt .Code 0 }}#cc2222{{ end }}
[TRACE] 21:32:32.322 text.go:Render({{ if gt .Code 0 }}#cc2222{{ end }}) - 8.127µs
[DEBUG] 21:32:32.322 text.go:patchTemplate:125 → {{ if gt .Code 0 }}#cc2222{{ end }}
[TRACE] 21:32:32.322 text.go:Render({{ if gt .Code 0 }}#cc2222{{ end }}) - 8.562µs
[DEBUG] 21:32:32.325 terminal_unix.go:TerminalWidth:85 → terminal width: 147
[TRACE] 21:32:32.325 terminal_unix.go:TerminalWidth() - 2.530018ms
[TRACE] 21:32:32.325 terminal.go:Shell() - 57ns
[TRACE] 21:32:32.325 terminal.go:Shell() - 23ns
[TRACE] 21:32:32.325 terminal.go:Flags() - 42ns
[DEBUG] 21:32:32.325 home.go:12 → /home/test
[TRACE] 21:32:32.325 terminal.go:GOOS() - 30ns
[DEBUG] 21:32:32.325 segment.go:Execute:113 → segment: Text
[DEBUG] 21:32:32.325 file.go:Get:75 → cache key not found: toggle_cache
[DEBUG] 21:32:32.325 segment.go:isToggled:234 → no toggles found
[TRACE] 21:32:32.325 text.go:Render(❯) - 233ns
[TRACE] 21:32:32.325 text.go:Render(plain) - 70ns
[TRACE] 21:32:32.325 terminal.go:Shell() - 140ns
[TRACE] 21:32:32.325 terminal.go:Shell() - 22ns
[TRACE] 21:32:32.325 terminal.go:Flags() - 46ns
[TRACE] 21:32:32.325 terminal.go:Shell() - 20ns
[TRACE] 21:32:32.325 terminal.go:Flags() - 22ns
[DEBUG] 21:32:32.325 home.go:12 → /home/test
[TRACE] 21:32:32.325 path.go:Path() - 12.061µs
[TRACE] 21:32:32.325 terminal.go:Flags() - 30ns
```

--- a/src/shell/scripts/omp.zsh
+++ b/src/shell/scripts/omp.zsh
@@ -147,8 +147,8 @@ function _omp_render_tooltip() {
     return
   fi
 
-  _omp_tooltip_command="$tooltip_command"
-  local tooltip=$(_omp_get_prompt tooltip --command="$tooltip_command")
+  _omp_tooltip_command="${tooltip_command}"
+  local tooltip=$(_omp_get_prompt tooltip --command="${tooltip_command}")
   if [[ -z $tooltip ]]; then
     return
   fi

